apps/web/app/api/(internal)/insights/lib/insights.ts

```
@@ -342,45 +345,49 @@
+      });
+      documentCache.revalidate({
         insightId: newInsight.id,
-      },
-    });
-    documentCache.revalidate({
-      insightId: newInsight.id,
-    });
+      });
+    }
+  } catch (error) {
+    throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)